stdlib/public/core/Arrays.swift.gyb

```diff
@@ -170,7 +170,7 @@ if True:
 """
   elif Self == 'Array':
     SelfDocComment = '''\
-/// An ordered, random-access collection.
+/// An ordered, random-access collection of elements of a single type.
```

Comment:
That's not accurate, is it?  They have a single static type, but of course you can have `[Any]` and then the types can be as diverse as you like.  I don't think it helps unless we're really trying to hide the fact that things like `[Any]` exist.